### DISCUSSION TITLE
Silent Hill: Downpour (and maybe other UE3 Titles)

### DISCUSSION THREAD

**Discussion by bomabomaboma** [xenia-canary/game-patches#11] [General]
I did some math and downcocked my GPU to roughly match the performance of a 1060 (still the most popular GPU).

Here is MSAA and FXAA enabled in Silent Hill: Downpour
(28 FPS)
![SHDownpourAA](https://user-images.githubusercontent.com/81534189/115763341-3a45c580-a359-11eb-90ad-8bfb37dbcf8c.png)

Here is all anti-aliasing disabled
(36 FPS)
MaxMultisamples=1 -> MaxMultisamples=0 in XeSHEngine.ini, [SystemSettings]          #2x MSAA to Disabled
ForceFXAA=1 -> ForceFXAA=0 in XeSHEngine.ini, [GlobalPostProcessSettings]          #Selfevident 
![SHDownpour-noAA](https://user-images.githubusercontent.com/81534189/115763800-b213f000-a359-11eb-91d3-f3a5bb545b44.png)

To me, the IQ looks nearly the same, even in motion. Only affects shadow edges.

Also, an anachronism for UE3 games in Xenia (I've observed this for all UE3 games so far)
bForceCPUAccessToGPUSkinVerts=False -> bForceCPUAccessToGPUSkinVerts=True ***Engine.ini, [SystemSettings] lowers CPU usage in Xenia without affecting performance or IQ, the exact opposite what it does in the PC platform counterparts.

**Comment by illusionyy** [comment]
Kinda sus, will look

**Reply by illusionyy** [reply]
Also I think @AphelionWasTaken made a patch to unlock fps but never released it, I should bug him again 😂

**Reply by bomabomaboma** [reply]
I've tried to find out what's causing water to not be properly rendered. 

It doesn't have anything to do with an error in UE3 occlusion culling.

**Reply by illusionyy** [reply]
> Also, an anachronism for UE3 games in Xenia (I've observed this for all UE3 games so far)
bForceCPUAccessToGPUSkinVerts=False -> bForceCPUAccessToGPUSkinVerts=True ***Engine.ini, [SystemSettings] lowers CPU usage in Xenia without affecting performance or IQ, the exact opposite what it does in the PC platform counterparts.

Any solid evidence behind this? I.e screenshots

**Reply by bomabomaboma** [reply]
False, default
![cpuvertfalse](https://user-images.githubusercontent.com/81534189/115978547-f9a2a380-a534-11eb-9db8-9ad1a017f26b.jpg)


True
![cpuvert](https://user-images.githubusercontent.com/81534189/115978559-0626fc00-a535-11eb-9dc2-c181c8f2c4a5.jpg)

Lowers top clock speeds, but increases usage on laptop Ryzen. This needs more investigating in UE3 games heavier on the CPU, with lots of SSAO and AI. Might end up as a dead-end (very plausible), or specific patches for laptop vs desktop users.

This game is hilarious. The developers were obviously unfamiliar with UE3; there are dozens of questionable, weird gaffes.
